pontoon/base/static/js/translate.js

```diff
@@ -3215,15 +3215,16 @@ var Pontoon = (function (my) {
 
       tags.forEach(function(tag) {
         var priority = '';
+        var slug = Pontoon.doNotRender(tag.slug);
```

Comment:
I'd add `var name = Pontoon.doNotRender(tag.name);` here and a comment on top of these two lines explaining why is that needed. Other than that, this is good to go. :)